backend_new/prisma/seed-helpers/application-preference-factory.ts

```diff
@@ -0,0 +1,37 @@
+import { MultiselectQuestions, Prisma } from '@prisma/client';
+import { randomNoun } from './word-generator';
+import { randomBoolean } from './boolean-generator';
+import { InputType } from '../../src/enums/shared/input-type-enum';
+
+export const preferenceFactory = (
+  multiselectQuestions: Partial<MultiselectQuestions>[],
+): Prisma.InputJsonValue => {
+  return JSON.stringify(
```

Comment:
I don't think we should be stringifying this. The preferences in the current DB are not saved as a string